code/src/pages/Seasons.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable linebreak-style */
+import React, { useState, useEffect } from 'react'
+import { useNavigate, useParams } from 'react-router-dom'
+import { TVSHOW_DETAILS_URL } from 'utils/Urls'
+import { Orbit } from '@uiball/loaders';
+
+export const Seasons = () => {
+  const [season, setSeason] = useState([])
+  const [loading, setLoading] = useState(false);
+  const navigate = useNavigate()
+  const { id } = useParams()
+
+  useEffect(() => {
+    setLoading(true)
+    fetch(TVSHOW_DETAILS_URL(id))
+      .then((res) => res.json())
+      .then((data) => setSeason(data.seasons))
+      .catch((error) => alert(error, 'error'))
+      .finally(() => setLoading(false));
+  }, [id])
+
+  const goBack = () => {
+    navigate(-1);
+  }
+
+  if (loading) {
+    return (
+      <div className="loading-container">
+        <h2>Loading..</h2>
+        <Orbit size={40} speed={0.9} color="#F9F9F9" />
+      </div>
+    );
+  }
+
+  return (
+    <section className="seasonsPage">
```

Comment:
Just one tiny thing in some of the season pages you can't see the back button because it is behind the image. But you went far and beyond with the project.